src/pe_reports/data/config.py

```diff
@@ -2,9 +2,10 @@
 
 # Standard Python Libraries
 from configparser import ConfigParser
-from importlib.resources import files
+import os
```

Comment:
@DJensen94 
If in the event that we wanted to use importlib, it would be`from  importlib_resources import files`. I remember the problems we had when this import requirement changed. 
@dav3r 
Do you have a recommendation as to which method is preferred, when reading in the `database.ini` file?

Reply:
I think it's best to use `importlib` if possible here.  Is there a reason that you moved away from it?

Reply:
@dav3r 
If i remember correctly there was some deprecation that took place in the importlib package and its a new package but implemented the same. Ok thanks for your input. We can get this changed back to using importlib.

Reply:
Are you maybe thinking of [this conversation](https://github.com/cisagov/pe-reports/pull/91#discussion_r739378886), where I recommended switching from `pkg_resources` to `importlib`?

Reply:
@dave, @cduhn17 issue addressed in the following commits [997c197](https://github.com/cisagov/pe-reports/pull/114/commits/997c197be90ee91fc5d24e635bc7c2c74fe82ee2), [9272975](https://github.com/cisagov/pe-reports/pull/114/commits/9272975ae126ab0bca525ba362493393ea22a6ac), [64b2290](https://github.com/cisagov/pe-reports/pull/114/commits/64b22908e1c9ed9b01799241a00a3a9761a41d78)

Reply:
@DJensen94 , 

Looks good and thanks for making those changes!